bitcoin/src/taproot/serialized_signature.rs

```diff
@@ -0,0 +1,304 @@
+//! Implements [`SerializedSignature`] and related types.
+//!
+//! Serialized Taproot signatures have the issue that they can have different lengths.
+//! We want to avoid using `Vec` since that would require allocations making the code slower and
+//! unable to run on platforms without allocator. We implement a special type to encapsulate
```

Comment:
Nit wording: unable to run on a platform without an allocator

Reply:
Why singular? There are multiple such platforms.

Reply:
> > Oops my bad!

Reply:
unable to run on platforms without an allocator